### DISCUSSION TITLE
Welcome to PowerBI-visuals-Gantt Discussions!

### DISCUSSION THREAD

**Discussion by toresenneseth** [Profitbase/PowerBI-visuals-Gantt#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by isazazaza** [comment]
Is there an option to purchase a single license that allows anyone who views the published reports to access them without needing an additional license?

**Reply by AndersAano** [reply]
Hi @isazazaza ,
You can purchase an enterprise license that will cover all users. 
That license will only make sense if you have about 1000+ users.

Best regards
Anders
